octobot_trading/data/order.py

```diff
@@ -284,6 +288,15 @@ def update_from_raw(self, raw_order):
             timestamp=get_value_or_default(raw_order, ExchangeConstantsOrderColumns.TIMESTAMP.value, None)
         )
 
+    def consider_as_filled(self):
```

Comment:
When we parse a trade received from exchange (historical trades), their are parsed using:
```python
def create_trade_instance_from_raw(trader, raw_trade):
    order = create_order_from_raw(trader, raw_trade)
    order.update_from_raw(raw_trade)
    return create_trade_from_order(order)
```
then ```create_trade_from_order(order)``` is creating the trade based on the order fields. Problem is that when call ```update_from_raw```, it won't set the order as filled, nor will set executed_prices etc since it doesn't know the order is actually filled (because created from a trade which is by definition a filled order).

We could also do this wiring in trade but I think is clearer to do it in order since the order filling engine is currently in ```Order```